cpu/schbench.py

```diff
@@ -22,9 +23,26 @@
 from avocado.utils import build, distro, git
 from avocado.utils.software_manager.manager import SoftwareManager
 
+DEFAULT_SCHBENCH_URL = 'https://git.kernel.org/pub/scm/linux/kernel/git/mason/schbench.git'
+
+# Define a dictionary with parameter names and default values
+DEFAULT_SCHBENCH_PARAMS = {
```

Comment:
same , please move this to test function, even can we have this yaml input so user should able to tweak param on the ply

Reply:
instead of having an additional constants (dict) to track this kept it as a part of self.params.get(.., defaut=**) similar to other tests.